### DISCUSSION TITLE
An unexpected error occurred during installation - Intune - macOS

### DISCUSSION THREAD

**Discussion by sviatoslav-kostetskyi** [mozilla/policy-templates#946] [General]
Hi!!!
Please help me, I can't install an extension that I have allowed.
I use Intune MDM for macOS devices.

	<key>EnterprisePoliciesEnabled</key>
	<true/>
	<key>DisablePasswordReveal</key>
	<true/>
	<key>PasswordManagerEnabled</key>
	<false/>
	<key>DisableSetDesktopBackground</key>
	<true/>
	<key>DontCheckDefaultBrowser</key>
	<true/>
	<key>ExtensionSettings</key>
	<dict>
		<key>*</key>
		<dict>
			<key>blocked_install_message</key>
			<string>Custom error message.</string>
			<key>install_sources</key>
			<array>
				<string>https://addons.mozilla.org/</string>
			</array>
			<key>installation_mode</key>
			<string>blocked</string>
		</dict>
		<key>{d634138d-c276-4fc8-924b-40a0ea21d284}</key>
		<dict>
			<key>installation_mode</key>
			 <string>force_installed</string>
			<key>install_url</key>
			<string>https://addons.mozilla.org/firefox/downloads/file/3972472/1password_x_password_manager-2.3.7.xpi</string>
		</dict>
		<key>https-everywhere@eff.org</key>
		<dict>
			<key>installation_mode</key>
			 <string>allowed</string>
		</dict>
	</dict>

<img width="1271" alt="Screenshot 2022-09-20 at 17 29 04" src="https://user-images.githubusercontent.com/101564850/191295146-0d8010f7-d37d-4dc5-97fa-4518735ff6cd.png">

**Comment by mkaply** [comment]
If you remove the

		"install_sources": [
			"https://addons.mozilla.org/*"
		],

part, it will work as expected.

install_sources is designed for adding additional sources for installs, you don't need to use it in your case where you're blocking all cases and allowing one ID for install.

In addition, if you use install_sources, it's a match pattern, so it would be something like "https://addons.mozilla.org/*" (although that still doesn't work. I'm looking into it)

**Comment by mkaply** [comment]
Fixing the pattern did work for me, so you can either remove it or change it to https://addons.mozilla.org/*

**Reply by sviatoslav-kostetskyi** [reply]
Thank you very much, everything works!
